addons/cds-connector-0.0.1/chart/cds-connector/templates/oauth-proxy-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "bundle.fullname" . }}
+  labels:
+    app: {{ template "bundle.fullname" . }}
+    component: {{ template "bundle.fullname" . }}
+type: Opaque
+data:
+  headers: {{ .Values.headers | b64enc }}
+  formparameters: {{ .Values.formparameters | b64enc }}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "bundle.fullname" . }}
+  labels:
+    app: {{ template "bundle.fullname" . }}
+    component: {{ template "bundle.fullname" . }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "bundle.fullname" . }}
+        component: {{ template "bundle.fullname" . }}
+      annotations:
+        sidecar.istio.io/inject: "true"
+    spec:
+      containers:
+      - name: {{ template "bundle.fullname" . }}
+        image: "jcawley5/kyma-oauth-proxy"
```

Comment:
What image is that? Please use official images only like the ones coming from kyma-incubator/connector-tools. Others might to support you in maintaining the connector and need a chance to change things.

Reply:
I can place the code for this image in the repo.